src/mappings/Tranche.ts

```diff
@@ -34,6 +34,6 @@ export function handleSupplyOrder(call: SupplyOrderCall): void {
   tk.save()
 
   let tokenBalance = loadOrCreateTokenBalance(account, token)
-  tokenBalance.pendingSupplyCurrency = amount
+  tokenBalance.pendingSupplyCurrency = tokenBalance.pendingSupplyCurrency.plus(amount)
```

Comment:
I don't understand why this would work. A supply order always updates the supply order value afaik, so if a new order comes along, it should overwrite the existing value, I think?

Reply:
E.g. if you first submit a supplyOrder of 10, and then a supplyOrder of 15, within the same epoch, the pendingSupplyCurrency will be 15, not 25.